tests/visualization/test_mpl.py

```diff
@@ -0,0 +1,194 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Based on the equivalent tests for the networkx matplotlib drawer:
+# https://github.com/networkx/networkx/blob/ead0e65bda59862e329f2e6f1da47919c6b07ca9/networkx/drawing/tests/test_pylab.py
+
+import os
+import unittest
+
+import retworkx
+from retworkx.visualization import mpl_draw
+
+try:
+    import matplotlib as mpl
+    import matplotlib.pyplot as plt
+
+    mpl.use("PS")
+    plt.rcParams["text.usetex"] = False
```

Comment:
Why it is needed to disable usetex?

Reply:
It's mostly for reproducibility in the tests, matplotlib's usetex mode relies on external latex installations to run: https://matplotlib.org/stable/tutorials/text/usetex.html and even if present can cause different behavior on different platforms or latex distributions. So just simplicity it makes sense to disable it here so things run consistently between different systems (although I don't think it should come up in any of the tests). 

That being said this line was just borrowed from networkx's equivalent drawer test module: https://github.com/networkx/networkx/blob/ead0e65bda59862e329f2e6f1da47919c6b07ca9/networkx/drawing/tests/test_pylab.py#L9